war/src/main/java/io/lumeer/engine/controller/CollectionMetadataFacade.java

```diff
@@ -333,6 +321,9 @@ public void dropAttribute(String collectionName, String attributeName) {
             dialect.concatFields(
                   LumeerConst.Collection.ATTRIBUTES_KEY,
                   attributePath(attributeName)));
+
+      throw new UnsupportedOperationException();
+      // TODO: drop children
```

Comment:
We just need to drop all attributes where the full name starts with the full name of the attribute which is being dropped. So probably a new filter similar to the wildcard filter will be needed.